resources/public/api/conf/1.0/schemas/retrieve_coding_out_underpayments_response.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve Coding Out underpayment and debt amounts",
+  "description": "Retrieve Coding Out underpayment and debt amounts",
+  "type": "object",
+  "properties": {
+    "source": {
+      "description": "Specifies the source of data.",
+      "type": "string",
+      "enum": [
+        "hmrcHeld",
+        "user"
+      ]
+    },
+    "selfAssessmentUnderPayments": {
+      "type": "array",
+      "description": "Array containing details of coding out Self Assessment underpayments.",
+      "items": {
+        "amount": {
+          "description": "The total coding out amount attributed to Self Assessment underpayments. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "type": "number",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "5000.99"
+        },
+        "relatedTaxYear": {
+          "description": "The tax year related to the underpayment.",
+          "type": "string",
+          "example": "2021-22"
+        },
+        "submittedOn": {
+          "description": "The date the coding out information was added. It is either the latest change date held within HMRC or the latest user submitted date. It is in the format YYYY-MM-DDThh:mm:ssZ.",
+          "type": "string",
+          "example": "2020-07-06T09:37:17Z"
+        }
+      }
+    },
```

Comment:
```suggestion
"selfAssessmentUnderPayments": {
      "type": "array",
      "description": "Array containing details of coding out Self Assessment underpayments.",
      "items": {
        "type": "object",
        "properties": {
          "amount": {
            "description": "The total coding out amount attributed to Self Assessment underpayments. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
            "type": "number",
            "multipleOf": 0.01,
            "minimum": 0.00,
            "maximum": 99999999999.99,
            "example": "5000.99"
          },
          "relatedTaxYear": {
            "description": "The tax year related to the underpayment.",
            "type": "string",
            "example": "2021-22"
          },
          "submittedOn": {
            "description": "The date the coding out information was added. It is either the latest change date held within HMRC or the latest user submitted date. It is in the format YYYY-MM-DDThh:mm:ssZ.",
            "type": "string",
            "example": "2020-07-06T09:37:17Z"
          }
        },
        "required": [
          "amount",
          "relatedTaxYear",
          "submittedOn"
        ]
      }
    },
```

For the array fields to show up properly in the RAML they need to be within a "properties" object like this. They should also be mandatory I think according to the spec. This applies to all of the arrays in this schema.